code/modules/mob/living/carbon/human/human_powers.dm

```diff
@@ -2092,6 +2189,45 @@
 
 		msg_admin_attack("[key_name_admin(src)] infected [key_name_admin(H)] with black k'ois! (<A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[src.x];Y=[src.y];Z=[src.z]'>JMP</a>)",ckey=key_name(src),ckey_target=key_name(H))
 
+/mob/living/carbon/human/proc/phalanx_transmit()
+	set name = "Phalanx Hivenet Transmission"
+	set desc = "Make a preset transmission over the Hivenet."
+	set category = "Hivenet"
+
+	var/obj/item/organ/internal/augment/language/vekatak/V = src.internal_organs_by_name[BP_AUG_LANGUAGE]
+	var/current_area = get_area(src)
+	var/list/messages = list(
+		"I remain operational.",
+		"Message recieved.",
+		"Order complete.",
+		"Cannot complete order.",
+		"I am on standby.",
+		"Minor damage registered.",
+		"Damage registered",
+		"Critical damage registered",
+		"Assistance required at [current_area].",
+		"Hostiles sighted at [current_area].",
+		"[current_area] is clear of threats."
```

Comment:
Requested changes:

Remove this, it's too powerful.

With the receiver, when kidnapped, blindfolded and transported somewhere, you can just say your location exactly, vs an actual vaurca would have no idea.

This would also show non-horizon areas, like away sites or ships, or the merc shuttle.

Reply:
Removed